scss/civicrm/search/pages/_results.scss

```diff
@@ -58,14 +60,26 @@
     margin: 0;
 
     > .element-right {
+      margin-right: 20px !important;
       margin-top: 7px;
     }
+  }
 
-    + .float-right {
-      float: none !important;
-      padding: 4px 10px 10px !important;
-      text-align: right;
-    }
+  .float-right {
```

Comment:
Can you show me what this is for?

Reply:
See below screenshot
![image](https://user-images.githubusercontent.com/26058635/36895213-20806676-1e34-11e8-808c-deb1e1004b27.png)